### PR TITLE
🔪 the old dropdown

### PR DESCRIPTION
**Pull request by lynnagara**

### CODE REVIEW

**Comment by getsentry-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">You should update CHANGES due to the size of this PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

**Comment by macqueen** [comment]
The react dropdown wasn't working for the discard dropdown (I think it's the only case where `LinkWithConfirmation` was the child), so I reverted to the old `DropdownLink` here: https://github.com/getsentry/sentry/pull/6591. Can you make sure that case works with these changes?

**Comment by lynnagara** [comment]
@macqueen Just confirming that this works with `keepMenuOpen=true`

**Comment by MaxBittker** [comment]
@lyn are you sure? I tried that and jess said it was supposed to redirect or something

**Comment by MaxBittker** [comment]
yeah, it's supposed to redirect to the issue stream

**Comment by lynnagara** [comment]
@MaxBittker Thanks! Sorry, it should be `alwaysRenderMenu=true`. Updated.

File: src/sentry/static/sentry/app/components/dropdownLink.jsx [hunk 1]
```diff
@@ -1,100 +1,220 @@
-import jQuery from 'jquery';
 import PropTypes from 'prop-types';
 import React from 'react';
 import classNames from 'classnames';
 
-require('bootstrap/js/dropdown');
-
-const DropdownLink = React.createClass({
-  propTypes: {
+class DropdownLink extends React.Component {
```
Comment (MaxBittker) [inline src/sentry/static/sentry/app/components/dropdownLink.jsx]: are the changes to this file copy paste from dropdownReact? or do they need to reviewed on their own merit?

Comment (lynnagara) [reply src/sentry/static/sentry/app/components/dropdownLink.jsx]: It's copy paste apart from a tiny change made in [a separate commit](https://github.com/getsentry/sentry/pull/6592/commits/5c4243df5442e6904e2b898224e1731d52eb96fe)

**Review by MaxBittker — APPROVED** [review]
I think we should be extra watchful after shipping this and watch out for cases where we're missing `alwaysRenderMenu` and cause behavior regressions, but the code looks good to me 👍 glad to be deleting that component!!!

**Comment by MaxBittker** [comment]
this should probably be shipped in the morning and we should watch out for issues/support tickets

**Comment by benvinegar** [comment]
💯

**Comment by MaxBittker** [comment]
^was this a sarcastic 💯

**Comment by benvinegar** [comment]
> ^was this a sarcastic 💯

No, I was trying to be supportive but maybe my choice of emoji was poor.

How about: 🤡

**Pull request merged**